app/assets/stylesheets/cart.css.scss

```diff
@@ -1,6 +1,6 @@
 .cart {
   background-image: 'texture.png';
-    background: oldlace;
+  background: oldlace;
```

Comment:
Color `oldlace` should be written in hexadecimal form as `#fdf5e6`<br>Color literals like `oldlace` should only be used in variable declarations; they should be referred to via variable everywhere else.